cli_li3ds/api.py

```diff
@@ -616,4 +618,6 @@ def update_obj(args, metadata, obj, type_):
 
 
 def isoformat(date):
+    if isinstance(date, str):
+        date = dateutil.parser.parse(date)
```

Comment:
Python has functions for parsing dates. See https://github.com/LI3DS/cli-li3ds/blob/fc65f28ad09cb1e340e61328bd59b13876e101f8/cli_li3ds/import_image.py#L187-L189 for example. Are you sure we need an extra dependency for that?

Reply:
Accuracy in minutes is not sufficient for json import of `_time` parameters.
I was looking to parse iso formatting dates and got [here](https://stackoverflow.com/questions/127803/how-to-parse-an-iso-8601-formatted-date), which hints at [dateutil](https://dateutil.readthedocs.io/en/stable/parser.html) or [iso8601](http://pyiso8601.readthedocs.io/en/latest/)